circleci.sh

```diff
@@ -133,10 +133,18 @@ publictests()
     make -f posix.mak -j$N publictests DUB=$DUB
 }
 
+# check modules for public unittests
+has_public_example()
+{
+    make -f posix.mak -j$N has_public_example DUB=$DUB
+}
+
 case $1 in
     install-deps) install_deps ;;
     setup-repos) setup_repos ;;
     coverage) coverage ;;
     publictests) publictests ;;
-    style) style ;;
+    has_public_example) has_public_example;;
+    style_lint) style_lint ;;
+    *) echo "Unknown command"; exit 1;;
```

Comment:
> IIRC, this was not the first time this happened.

Yep ...

> Any ideas on how to improve the situation and avoid this from happening again?

This line should at least prevent the same error in the future.
Not really sure on a general solution - I guess that no ones looks at CircleCi doesn't help ...

> Was there an error being ignored in the test script somewhere perhaps?

[An unknown command was called https://circleci.com/gh/dlang/phobos/3042) and thus nothing was executed.